src/main/java/uk/gov/companieshouse/mapper/CompanyOfficerMapper.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.companieshouse.mapper;
+
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.api.model.officers.CompanyOfficerApi;
+import uk.gov.companieshouse.model.db.dissolution.DissolutionDirector;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class CompanyOfficerMapper {
```

Comment:
This is not a mapper. Mappers a dumb classes that do no calculations. This is a service, or calculator, or validator.

Reply:
Should this be moved to the CompanyOfficerService class?

Reply:
If the other code in that class is moved out to a client class then potentially yes. You could alternatively just change the name and location of this file to something like `CompanyOfficerMajorityValidator` or something.

Reply:
Not a mapper, more of a CompanyOfficerChecker or CompanyOfficerValidator or something.